processor/deltatocumulativeprocessor/README.md

```diff
@@ -25,6 +25,8 @@ processors:
     deltatocumulative:
         # how long until a series not receiving new samples is removed
         [ max_stale: <duration> | default = 5m ]
+ 
+        # upper limit of streams to track. new streams exceeding this limit
+        # will be dropped
+        [ max_streams: <int> | default = off]
```

Comment:
The default is `0`, correct? (which is interpreted as off). Should we notate that in the docs?